app/policies/application_policy.rb

```diff
@@ -0,0 +1,54 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    scope.where(:id => record.id).exists?
+  end
+
+  def create?
+    user.present?
+  end
+
+  def new?
+    create?
+  end
+
+  def update?
+    user.present?
+  end
+
+  def edit?
+    update?
+  end
+
+  def destroy?
+    false
```

Comment:
Was supposed to add this to the wiki_policy but see I didn't - will add in current branch and merge